packages/mobile/src/components/share-drawer/ShareToStorySticker.tsx

```diff
@@ -0,0 +1,111 @@
+import type { Track, User } from '@audius/common'
+import type { StyleProp, ViewStyle } from 'react-native'
+import { Image, View } from 'react-native'
+
+import AudiusLogo from 'app/assets/images/audiusLogoHorizontal.svg'
+import { Divider, Text } from 'app/components/core'
```

Comment:
oh cool so we have the share-to-story-sticker subcomponent, yeah i wonder how we might isolate all the share-story logic in share-drawer.... i guess in a hook or smtg?

Reply:
i guess you could have all the share-logic in this component, and render null until user has clicked "share to instagram"?